src/ctype/CTypeMetadata.spec.ts

```diff
@@ -0,0 +1,46 @@
+import CType from './CType'
+import ICType from '../types/CType'
+import CTypeMetadata from './CTypeMetadata'
+import * as CTypeUtils from './CTypeUtils'
+import { CTypeWrapperMetadata } from './CTypeSchema'
+import ICTypeMetadata from '../types/CTypeMetadata'
+
+jest.mock('../blockchainApiConnection/BlockchainApiConnection')
+
+describe('CType', () => {
+  const ctype = new CType({
+    schema: {
+      $id: 'http://example.com/ctype-1',
+      $schema: 'http://kilt-protocol.org/draft-01/ctype#',
+      properties: {
+        'first-property': { type: 'integer' },
+        'second-property': { type: 'string' },
+      },
+      type: 'object',
+    },
+  } as ICType)
+
+  const ctypeHash = ctype.hash
+
+  const ctypeMetadata = {
+    title: { default: 'string' },
```

Comment:
Why are you using `'string'` as the title? I feel. that you are confused about how the metadata looks like and the json-schema.

Reply:
Understood now.